platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerView.swift

```diff
@@ -21,25 +21,17 @@ import SwiftUI
 public struct WysiwygComposerView: UIViewRepresentable {
     // MARK: - Internal
 
-    public var content: WysiwygComposerContent
-    public var replaceText: (NSAttributedString, NSRange, String) -> Bool
-    public var select: (NSAttributedString, NSRange) -> Void
-    public var didUpdateText: (UITextView) -> Void
+    public var viewModel: WysiwygComposerViewModel
```

Comment:
Should we move this to a protocol including all functions that are actually relevant for the UITextView ? Right now it will have access to a bunch of functions that are not necessarily useful from the UITextView perspective

Reply:
Good idea, will work on that!

Reply:
Added the protocol